### PR TITLE
[#17] game mod name

### PR DESCRIPTION
**Pull request by budnickip**

### CODE REVIEW

File: styles/components/gameMode.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.question{
+    width: 100%;
+    background-color: #ffffff;
+    color: #000000;
+    padding: 18px 0px;
+    font-weight: 600;
+    font-size: 1.5em;
+    display: flex;
+    justify-content: center;
+    border-radius: 16px;
```
Comment (mfrydrychowicz) [inline styles/components/gameMode.scss:10]: not sure how sass works, but can you make .box class in App.css that will have common style shared between all boxes? 
also use variables defined in :root ?

**Review by mfrydrychowicz — CHANGES_REQUESTED** [review]

File: styles/components/gameMode.scss (hunk 1 above)
Comment (mfrydrychowicz) [reply styles/components/gameMode.scss:10]: this is more like changes to consider. If you say "no" i'm ok with that

File: test/gameMode.spec.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    `
+    const text = 'Test question'
+    gameMode(text)
+    expect(screen.getByTestId('question')).toHaveTextContent(text)
+})
+/*
+test("Question shouldn't be empty",()=>{
+    const question = gameMode('Not Empty string')
+    expect(question).toBe(true)
+}) */
```
Comment (mfrydrychowicz) [inline test/gameMode.spec.js:30]: leftovers?

File: styles/components/gameMode.scss (hunk 1 above)
Comment (budnickip) [reply styles/components/gameMode.scss:10]: sass works exactly the same like css, so yes, we should do this in our project. I will add this tomorrow :).

**Pull request closed without merging**